pyrit/score/look_back_scorer.py

```diff
@@ -10,19 +10,22 @@
 from pyrit.models.score import Score, UnvalidatedScore
 from pyrit.prompt_target import PromptChatTarget
 from pyrit.score.scorer import Scorer
+import re
 
 
 class LookBackScorer(Scorer):
     """
     Create a score from analyzing the entire conversation and adds them to the database.
 
     Parameters:
+        exclude_instruction_prompts (bool): If True, excludes instruction prompts from the conversation.
         chat_target (PromptChatTarget): The chat target to use for scoring.
     """
 
-    def __init__(self, chat_target: PromptChatTarget) -> None:
+    def __init__(self, exclude_instruction_prompts: bool, chat_target: PromptChatTarget) -> None:
```

Comment:
```suggestion
    def __init__(self, chat_target: PromptChatTarget, *, exclude_instruction_prompts: bool) -> None:
```
This makes it keyword-only

Reply:
More broadly, I'm wondering if it should just be an `exclude_regex` that we match to prompts? So if it's not set then we include everything. @rlundeen2 any thoughts?